pro/src/app/AppRouter/routesMap.tsx

```diff
@@ -73,16 +87,35 @@ export const routes: RouteConfig[] = [
     lazy: () => import('pages/Desk/Desk'),
     path: '/guichet',
     title: 'Guichet',
+    meta: {
+      siteMap: {
+        parent: 'Individuel',
```

Comment:
Idem, c'est de la complexité qui me semble pas nécessaire vs rajouter un lien bateau 1 fois / mois
Là j'ai envie de dire Keep It Simple and Stupid :smile: